gridscale/resource_gridscale_postgressql.go

```diff
@@ -0,0 +1,386 @@
+				Type:         schema.TypeString,
+				Description:  "The human-readable name of the object. It supports the full UTF-8 character set, with a maximum of 64 characters.",
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			"release_no": {
+				Type:         schema.TypeString,
+				Description:  "Release no. of postgresSQL service.",
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			"performance_class": {
+				Type:        schema.TypeString,
+				Description: "Performance class of postgresSQL service.",
+				Required:    true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					valid := false
+					for _, class := range postgresPerformanceClass {
+						if v.(string) == class {
+							valid = true
+							break
+						}
+					}
+					if !valid {
+						errors = append(errors, fmt.Errorf("%v is not a valid postgres performance class. Valid values are: %v", v.(string), strings.Join(postgresPerformanceClass, ",")))
+					}
+					return
+				},
+			},
+			"username": {
+				Type:        schema.TypeString,
+				Description: "Username for PostgresSQL service.",
+				Computed:    true,
+			},
+			"password": {
+				Type:        schema.TypeString,
+				Description: "Password for PostgresSQL service.",
+				Computed:    true,
+			},
+			"listen_port": {
+				Type:        schema.TypeSet,
+				Description: "Ports that PostgresSQL service listens to.",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"port": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"security_zone_uuid": {
+				Type:        schema.TypeString,
+				Description: "Security zone UUID linked to PostgresSQL service.",
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+			},
+			"network_uuid": {
+				Type:        schema.TypeString,
+				Description: "Network UUID containing security zone.",
+				Computed:    true,
+			},
+			"service_template_uuid": {
+				Type:        schema.TypeString,
+				Description: "PaaS service template that postgresSQL service uses.",
+				Computed:    true,
+			},
+			"usage_in_minutes": {
+				Type:        schema.TypeInt,
+				Description: "Number of minutes that PostgresSQL service is in use",
+				Computed:    true,
+			},
+			"change_time": {
+				Type:        schema.TypeString,
+				Description: "Time of the last change.",
+				Computed:    true,
+			},
+			"create_time": {
+				Type:        schema.TypeString,
+				Description: "Time of the creation.",
+				Computed:    true,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Description: "Current status of PostgresSQL service.",
+				Computed:    true,
+			},
+			"max_core_count": {
+				Type:        schema.TypeInt,
+				Description: "Maximum CPU core count. It is used for autoscaling.",
```

Comment:
We should clarify this a bit more (possibly in the docs). What does this value mean?